### DISCUSSION TITLE
Image stabilization

### DISCUSSION THREAD

**Discussion by vvsvvsvvs** [rockchip-linux/mpp#558] [Q&A]
When I take a look at the TRM docs by Rock-chips, I can found the "Video stabilization" block started with 32** chips. I can see the structure in mpp API with the corresponding field. But how to turn stabilization ON? It would be good to have these feature  for computer vision, reducing noise and signal math requirements.
![Videostab](https://github.com/rockchip-linux/mpp/assets/11758975/88067a35-cb50-4f38-ae24-a0692a7ae06c)

**Comment by HermanChen** [comment]
Its effect is very limited and only on vepu1/2 which performance is only 1080p@30fps.
Also it depends on previous stage mv infomation which is hard to get in most cases.
So we do not use it in pratical.

**Comment by vvsvvsvvs** [comment]
rather on newer chips? like RK35**?
hm....OK...
